docs/topics/templates/document-attributes.adoc

```diff
@@ -83,7 +83,7 @@
 :ProductDocCoreGuideURL: https://github.com/windup/windup/wiki/Core-Development-Guide
 
 // URL for downloads on developers.redhat.com
-:ProductDownloadURL: https://developers.redhat.com/products/
+:DownloadPageURL: https://developers.redhat.com/products/mta/download
 
```

Comment:
Did you remove all download URLs that pointed to specific products on the developers.redhat.com site?

Reply:
@apinnick: Created the following attributes, to cover all current download procedures:

:MTADownloadPageURL: https://developers.redhat.com/products/mta/download
:CodeReadyStudioDownloadPageURL: https://developers.redhat.com/products/codeready-studio/download/

Updated the relevant URLs as well as the procedure texts: 

http://file.emea.redhat.com/rhoch/01032021/webcg/html-single/#installing_the_web_console
http://file.emea.redhat.com/rhoch/01032021/pluging/html-single/#installing-web-console_plugin-guide

Please review.